guides/common/modules/proc_exporting-a-content-view-version-syncable.adoc

```diff
@@ -0,0 +1,83 @@
+[id="Exporting_a_Content_View_Version_Syncable_{context}"]
+= Exporting a Content View Version in a Syncable Format
+
+You can export a version of a Content View to an archive file from {ProjectServer} in a syncable format.
+
+ifdef::satellite[]
+You can then serve the generated content via a local webserver on the importing  {ProjectServer} or in another {ProjectServer} organization.
+
+You cannot directly import Syncable Format exports. Instead on the importing {ProjectServer} you would need to
+* Copy the generated content to an HTTP/HTTPS webserver that is accessible to importing {ProjectServer}
+* Update your CDN configuration to *Custom CDN*
+* Set the CDN URL to point to the web server
+* Optionally set a SSL CA Credential if the web server requires it
+* Enable the Repository
+* Synchronize the Repository.
+endif::[]
+
+ifndef::satellite[]
+You can use the generated content to create the same repository in another {ProjectServer} or in another {ProjectServer} organization via content import.
+On importing the exported archive, a regular Content View is created or updated on your downstream {ProjectServer}.
+endif::[]
```

Comment:
```suggestion
For more information, see xref:Importing_a_Content_View_Version_{context}[].
endif::[]
```

Reply:
FYI, there will be a special module for syncable import.